client/wc-api/reports/items/operations.js

```diff
@@ -0,0 +1,74 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * WooCommerce dependencies
+ */
+import { stringifyQuery } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import { getResourceIdentifier, getResourcePrefix } from '../../utils';
+import { NAMESPACE } from '../../constants';
+import { SWAGGERNAMESPACE } from 'store/constants';
+
+// TODO: Remove once swagger endpoints are phased out.
+const swaggerEndpoints = [ 'categories', 'coupons', 'taxes' ];
+
+const typeEndpointMap = {
```

Comment:
Given that we are using this for items and stats, could it be extracted to a `/utils.js` file so we avoid repeating it?

Reply:
Oh right, didn't notice that. :sweat_smile: